api/test/wfl/tools/workloads.clj

```diff
@@ -215,3 +269,9 @@
 (defn append-to-workload! [samples workload]
   (jdbc/with-db-transaction [tx (fixtures/testing-db-config)]
     (aou/append-to-workload! tx samples workload)))
+
+(defmulti  postcheck
```

Comment:
Where are you implementing this?
Might this be context specific, i.e. you'll want to verify some condition based upon what exactly you're testing?

Reply:
There is a :default implementation and one for SG now.
Yes.  It depends on what the module is responsible for.